src/app/ui/components/Account/AccountDetails/AccountDetails.py

```
@@ -45,6 +49,38 @@ def create_widgets(self):
+            }
+            QPushButton:hover {
+                background-color: #3e3e3e;
+            }
+        """)
+        login_button.setFixedHeight(50)
+
+        def login_button_funcion():
+            try:
+                device = uiautomator2.connect()
+                login(device=device, email=self.account["email"], password=self.account["password"])
+            except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)